apps/docs/src/app/core/component-docs/shellbar/examples/shellbar-user-actions-menu-compact-example.component.html

```diff
@@ -0,0 +1,106 @@
+                <li fd-list-item fd-user-actions-menu-item>
+                    <a fd-list-link>
+                        <i fd-list-icon glyph="border"></i>
+                        <span fd-list-title>
+                            Lorem Ipsum is simply dummy text of the printing and typesetting industry
+                        </span>
+                    </a>
+                </li>
+                <li fd-list-item fd-user-actions-menu-item>
+                    <a fd-list-link [routerLink]="['/']">
+                        <i fd-list-icon glyph="chain-link"></i>
+                        <span fd-list-title> Router link </span>
+                    </a>
+                </li>
+                <li fd-list-item fd-user-actions-menu-item [submenu]="subMenuLevel2">
+                    <a fd-list-link navigationIndicator="true" (click)="clickUserAction($event)">
+                        <i fd-list-icon glyph="example"></i>
+                        <span fd-list-title> Example click </span>
+                    </a>
+                </li>
+                <li fd-list-item fd-user-actions-menu-item [dialogContent]="aboutTemplate">
+                    <a fd-list-link>
+                        <i fd-list-icon glyph="hint"></i>
+                        <span fd-list-title> About </span>
+                    </a>
+                </li>
+
+                <li fd-list-item fd-user-actions-menu-item *ngFor="let item of userMenuListItems">
+                    <a fd-list-link (click)="item.callback && item.callback()">
+                        <i fd-list-icon [glyph]="item.glyph"></i>
+                        <span fd-list-title> {{ item.text }} </span>
+                    </a>
+                </li>
+            </ul>
+
+            <fd-user-actions-menu-footer align="right">
+                <fd-button-bar [compact]="true" label="Sign Out" fdType="transparent"></fd-button-bar>
+            </fd-user-actions-menu-footer>
+        </fd-user-actions-menu>
+    </fd-shellbar-actions>
+</fd-shellbar>
+
+<!-- Example dialog for user actions menu -->
+<ng-template let-dialog let-dialogConfig="dialogConfig" #aboutTemplate>
+    <fd-dialog [dialogConfig]="dialogConfig" [dialogRef]="dialog">
+        <fd-dialog-header>
+            <h1 fd-title>About</h1>
+        </fd-dialog-header>
+
+        <fd-dialog-body> About dialog for user actions menu </fd-dialog-body>
+        <fd-dialog-footer>
+            <fd-button-bar fdType="transparent" label="OK" (click)="dialog.close()"> </fd-button-bar>
+        </fd-dialog-footer>
+    </fd-dialog>
+</ng-template>
+
+<fd-user-actions-submenu #subMenuLevel2 title="SubMenu level 2">
+    <ul fd-list [noBorder]="true" [compact]="true" navigationIndicator="true">
+        <li fd-list-item fd-user-actions-menu-item>
+            <a fd-list-link>
+                <i fd-list-icon glyph="border"></i>
+                <span fd-list-title> Lorem Ipsum is simply dummy text of the printing and typesetting industry </span>
```

Comment:
hovering over this long text should show the entire text in a tooltip as it appears truncated.